hash/sega_beena_cart.xml

```diff
@@ -681,29 +1845,77 @@ license:CC0-1.0
 			</dataarea>
-			<dataarea name="page1" size="0x1b01d31"> <!-- book pages -->
-				<rom name="0 - 0001.png" size="0x1b01d31" crc="64bd8b70" sha1="13093c55209e9bc5aae9074c11b0f20b10ba27c2"/>
+			<dataarea name="page1" size="0x01b01d31"> <!-- book pages -->
+				<rom name="0 - 0001.png" size="0x01b01d31" crc="64bd8b70" sha1="13093c55209e9bc5aae9074c11b0f20b10ba27c2"/>
+			</dataarea>
+			<dataarea name="page2" size="0x01c7c875"> <!-- book pages -->
+				<rom name="0 - 0002.png" size="0x01c7c875" crc="ffbf7001" sha1="a8c46fafb745aae29540e9438c114df24953b644"/>
+			</dataarea>
+			<dataarea name="page3" size="0x01d6c05a"> <!-- book pages -->
+				<rom name="0 - 0003.png" size="0x01d6c05a" crc="33ad5c4e" sha1="7c6eb6bd0552fa118cb2535f05a082ce9e621ced"/>
+			</dataarea>
+			<dataarea name="page4" size="0x015f2613"> <!-- book pages -->
+				<rom name="0 - 0004.png" size="0x015f2613" crc="f1efe876" sha1="2fffe5aa9d75a4ecaa62b225a9c1489bfb567443"/>
 			</dataarea>
-			<dataarea name="page2" size="0x1c7c875"> <!-- book pages -->
-				<rom name="0 - 0002.png" size="0x1c7c875" crc="ffbf7001" sha1="a8c46fafb745aae29540e9438c114df24953b644"/>
+			<dataarea name="page5" size="0x01a22cc7"> <!-- book pages -->
+				<rom name="0 - 0005.png" size="0x01a22cc7" crc="d5d69788" sha1="72c456feaa43fc6da61e67b75f2de9825f002da6"/>
 			</dataarea>
-			<dataarea name="page3" size="0x1d6c05a"> <!-- book pages -->
-				<rom name="0 - 0003.png" size="0x1d6c05a" crc="33ad5c4e" sha1="7c6eb6bd0552fa118cb2535f05a082ce9e621ced"/>
+			<dataarea name="page6" size="0x01d487f1"> <!-- book pages -->
+				<rom name="0 - 0006.png" size="0x01d487f1" crc="cd0163ed" sha1="24bf937e2295b4718b7df775a2f1e2ba5648b95c"/>
 			</dataarea>
-			<dataarea name="page4" size="0x15f2613"> <!-- book pages -->
-				<rom name="0 - 0004.png" size="0x15f2613" crc="f1efe876" sha1="2fffe5aa9d75a4ecaa62b225a9c1489bfb567443"/>
+			<dataarea name="page7" size="0x01da9a29"> <!-- book pages -->
+				<rom name="0 - 0007.png" size="0x01da9a29" crc="7efb76c2" sha1="3b9c474dece679d6b073cffb31d0b641bf5c34f9"/>
 			</dataarea>
-			<dataarea name="page5" size="0x1a22cc7"> <!-- book pages -->
-				<rom name="0 - 0005.png" size="0x1a22cc7" crc="d5d69788" sha1="72c456feaa43fc6da61e67b75f2de9825f002da6"/>
+			<dataarea name="page8" size="0x01a55c9e"> <!-- book pages -->
+				<rom name="0 - 0008.png" size="0x01a55c9e" crc="c5ebe3a8" sha1="34e3f80757a8fbeef8f20110c585b31ecadeb891"/>
 			</dataarea>
-			<dataarea name="page6" size="0x1d487f1"> <!-- book pages -->
-				<rom name="0 - 0006.png" size="0x1d487f1" crc="cd0163ed" sha1="24bf937e2295b4718b7df775a2f1e2ba5648b95c"/>
+		</part>
+	</software>
+
+	<software name="pokebat" supported="no">
+		<description>Pocket Monsters Advanced Generation Pokémon Suuji Battle!!</description>
+		<year>2005</year>
+		<publisher>Sega Toys</publisher>
+		<info name="alt_title" value="ポケットモンスターアドバンスジェネレーションポケモンすうじバトル!!"/>
```

Comment:
The Japanese literally says “Pocket Monster Advance Generation” – is there actual English text showing “Monsters” and “Advanced”?